app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = get_user_id params[:user_id]
-    @posts = @user.posts.includes((:comments))
+    @posts = @user.posts
   end
```

Comment:
- [ ] Kindly solve the  N+1 problem when fetching all posts and their comments for a user.

   - [HINT] In the `posts#index` actions, you can use:
   ```
   @posts = @user.posts.includes(:comments)
   ```
   instead of 
   ```
   @posts = @user.posts
   ```